spec.html

```diff
@@ -47128,7 +47128,7 @@ <h1>WeakRef.prototype</h1>
 
     <emu-clause id="sec-properties-of-the-weak-ref-prototype-object">
       <h1>Properties of the WeakRef Prototype Object</h1>
-      <p>The <dfn>WeakRef prototype</dfn> object:</p>
+      <p>The <dfn>WeakRef prototype object</dfn>:</p>
```

Comment:
Same as below but in https://tc39.es/ecma262/#sec-properties-of-weak-ref-instances.